packages/p2p/src/components/my-profile/my-profile.jsx

```diff
@@ -1,9 +1,95 @@
-import React from 'react';
-import './my-profile.scss';
+import React        from 'react';
+import PropTypes    from 'prop-types';
 import { localize } from 'deriv-translations';
+import                   './my-profile.scss';
+
+// TODO: This is just mockup data. it will be removed after Adding BE API.
+const user_data = {
+    first_name                : 'johny',
+    last_name                 : 'bravo',
+    security_deposit          : 0,
+    buy_trades                : 23,
+    sell_trades               : 25,
+    past_month_trades         : 42,
+    past_month_completion_rate: 99,
+    average_release_time      : 5,
+    average_release_unit      : 'min',
+};
+
+function capitalize (string) {
```

Comment:
how about a different naming?
```suggestion
function toSentenceCase(string) {
```

Reply:
also if it's a utility function it should be in the utility files and imported here instead